src/vs/editor/contrib/suggest/browser/suggestWidget.ts

```diff
@@ -760,6 +761,7 @@ export class SuggestWidget implements IDisposable {
 		if (position === null) {
 			if (this._isDetailsVisible()) {
 				this._details.hide(); //todo@jrieken soft-hide
+				this._list.ariaDescribedBy = '';
```

Comment:
Do we need something similar in `toggleDetails`? E.g when showing suggest, without further updating the items list, I can show/hide the details via `ctrl+space`.